scripts/create-docs.py

```diff
@@ -0,0 +1,42 @@
+import os
+import re
+import sys
+
+lib_dir = "lib"
+output_resource = sys.argv[1] if len(sys.argv) > 1 else "all-keywords.robot"
+
+# Collect all .robot files
+robot_files = ["keywords.robot"]
+for root, dirs, files in os.walk(lib_dir):
+    for file in files:
+        if file.endswith(".robot"):
+            robot_files.append(os.path.join(root, file))
+
+# Regex pattern to detect keyword definitions
+keyword_pattern = re.compile(
+    r"\n(?:(?!Library|Resource|Variables|Documentation)[A-Z][a-z0-9_]+[\ ]+.*)"
```

Comment:
```suggestion
    r"^(?:(?!Library|Resource|Variables|Documentation)[A-Z][a-z0-9_]+[\ ]+.*)",
    r.MULTILINE
```

Reply:
Fixed in 2774df89044aa7b0999e4f32309836c74e45a346